### ISSUE TITLE
`backgroundColor` option not working when setting the backgroundColor on the canvas

### ISSUE THREAD

**Issue by davidhatten** [exupero/saveSvgAsPng#184]
Issue: When I use the `backgroundColor` option, it is not being recognized.

Workaround: set the backgroundColor style on the canvas before sending it to the library

Description:

I have an svg canvas with the 'backgroundColor' option set to 'white'. When I attempt to export the canvas with no options, the background is white. When I attempt to use the option `{backgroundColor: 'transparent'}`, the exported background is still white.

```javascript
exportCanvas = () => {
        const canvas = document.getElementById(`drawingCanvas`);
        saveSvgAsPng(canvas, `geometry.png`, {backgroundColor: `transparent`});
    }
```
```javascript
render() {
        const svgStyle = {
            borderWidth: `1px`,
            borderStyle: `solid`,
            display: `block`,
            margin: `auto`,
            backgroundColor: `white`,
        };
       
        ...

        return (
            <div>
                <svg
                    id="drawingCanvas"
                    viewBox={`0 0 ${this.props.svgWidth} ${this.props.svgHeight}`}
                    width="95%"
                    height="95%"
                    style={svgStyle}
                    onClick={this.initiateDraw}>
                    You must use a browser that supports HTML5.
                    {drawnShapes}
                </svg>
            </div>
        );
```

This code works and achieves what I want.

```javascript
exportCanvas = () => {
    const canvas = document.getElementById(`drawingCanvas`);
    canvas.style.backgroundColor = `transparent`;
    saveSvgAsPng(canvas, `geometry.png`);
    canvas.style.backgroundColor = `white`;
}
```

I've tried going into the library and forcing it to set style options on its own canvas object as it exports it with no luck.

**Comment by exupero** [comment]
I've published v1.4.6 to address this. Background color was implemented by drawing a rectangle the size of the image onto the canvas before transferring the SVG over, so it was purely additive and could not unset any background colors on your SVG. I like your method better. Setting the SVG's background color via CSS is simpler, and also allows unsetting the background color, as you're trying to do.

If you encounter any problems, feel free to re-open this issue.

**Comment by davidhatten** [comment]
The new change works exactly as anticipated. Thank you!
